.eslintrc.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
     parser: '@typescript-eslint/parser',
     root: true,
-    plugins: ['node'],
+    plugins: ['node', 'prettier'],
```

Comment:
My IDE was complaining it couldn't find `eslint-plugin-prettier` (we are using it inside `extends`). I added it to the plugins and here.

Reply:
Gotcha, thanks for the PR. I was hesitating to add linting to the `lib` folder until we were at a stable point, but I think it's time